app/routes/movies/$movieId/index.tsx

```diff
@@ -24,21 +26,27 @@ export const loader: LoaderFunction = async ({ params }) => {
   const similar = await getSimilar('movie', mid);
   const videos = await getVideos('movie', mid);
   const credits = await getCredits('movie', mid);
+  const recommendations = await getRecommendation('movie', mid);
 
-  if (!similar || !videos || !credits) throw new Response('Not Found', { status: 404 });
+  if (!similar || !videos || !credits || !recommendations)
+    throw new Response('Not Found', { status: 404 });
 
   return json<LoaderData>({
     videos,
-    credits,
     similar,
+    recommendations,
+    topBilledCast: credits && credits.cast && credits.cast.slice(0, 9),
```

Comment:
@ctuanle ở đây mình có thể return về data mình cần được nơi m 🤟

Reply:
@Khanhtran47 return gì k hiểu nơi, chỗ lấy data từ routes parent á ?

Reply:
@ctuanle kiểu cái m nói chỉ lấy data cần thiết như bên graphQL ý

Reply:
@ctuanle m đọc cái bài ni cho dễ hiểu hơn 😅
https://sergiodxa.com/articles/load-only-the-data-you-need-in-remix

Reply:
@Khanhtran47 t nhớ chỗ loader đúng là mình return cái gì cũng được thật 😅